### PR TITLE
Add s390x support for travis

### PR DESCRIPTION
**Pull request by Nayana-ibm**
As Travis CI [officially supports](https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z) s390x builds, adding support for same.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,15 +21,32 @@ install:
...
 - . scripts/travis-env.sh && mkdir build && cd build && ../autogen.sh --enable-werror && make && make check
 matrix:
   include:
   - name: "linux"
     os: linux
+    dist: trusty
   - name: "osx"
     os: osx
     osx_image: xcode9.3
+  - name: "linux"
```
Comment (lsh123) [inline .travis.yml]: May be changed it to "linux-s390x"?

File: .travis.yml [hunk 2]
```diff
@@ -16,15 +21,32 @@ install:
...
+      sudo apt-get install libgnutls-dev; 
+    else 
+      sudo apt-get install libgnutls28 libgnutls-dev; 
+    fi;
+  fi
 script:
 - . scripts/travis-env.sh && mkdir build && cd build && ../autogen.sh --enable-werror && make && make check
 matrix:
   include:
   - name: "linux"
```
Comment (lsh123) [inline .travis.yml]: May be change it to "linux-x86_64"?

File: .travis.yml (hunk 1 above)
Comment (Nayana-ibm) [reply .travis.yml]: Incorporated review comments

**Comment by Nayana-ibm** [comment]
Incorporated review comments . Please review

**Comment by lsh123** [comment]
Thank you!

**Pull request merged**